### PR TITLE
Release v6.5 lacework-agent Helm Charts

### PR DESCRIPTION
**Pull request by brooks-connor-a**

### CODE REVIEW

File: lacework-agent/values.yaml [hunk 1]
```diff
@@ -87,6 +97,18 @@ laceworkConfig:
   # [Optional] Kubernetes cluster name
   # https://support.lacework.com/hc/en-us/articles/360005263034-Deploy-on-Kubernetes
   kubernetesCluster:
+  # [Optional] Define the behavior of the agent's package scans
+  packagescan:
```
Comment (c-p-i-o) [inline lacework-agent/values.yaml:101]: Why did `packagescan` appear twice in this file?
Seems like a bug to me.

Comment (nschmeller) [reply lacework-agent/values.yaml:101]: That's correct @c-p-i-o. @brooks-connor-a can you check in the changes from 243a640473dff4ae9d4ffd7b9abc29b675bf7770 and 243a640473dff4ae9d4ffd7b9abc29b675bf7770 in the agent repo?

Comment (brooks-connor-a) [reply lacework-agent/values.yaml:101]: @nschmeller, that's the same commit hash twice. I do see the fix that went in two days ago, in main. I'll update this PR with the fix, and let me know if theres a second change needed as well

Comment (nschmeller) [reply lacework-agent/values.yaml:101]: Yes, sorry, was on mobile :) that's the one, minus the properties test change

Comment (brooks-connor-a) [reply lacework-agent/values.yaml:101]: ✅

**Review by nschmeller — APPROVED** [review]

**Pull request merged**